### ISSUE TITLE
The JS SDK implements the new versioned HTTP API

### ISSUE THREAD

**Issue by brunocalza** [tablelandnetwork/js-tableland#232]

**Comment by jsign** [comment]
@joewagner, I assigned this to you but let me know if should be anyone else.

Technically speaking, with the spec, the SDK changes can be done in parallel, but probably next week I'll have the validator ready if that also helps to doing & test the new SDK changes. I'll keep you posted.

**Comment by jsign** [comment]
@joewagner, I'll update the Milestone for this. In our product guild call today we pushed back the deployment with this new HTTP API for January, so there will be more time. Mentioning this now since I also noticed you'll be OOO soon... just to signal there will be time.

**Comment by joewagner** [comment]
Catching up here... This all looks great!  Carson has already done some (most) of the work for this, but I'll happily take DRI and make sure everything gets done.
